README.md

```diff
@@ -94,8 +101,6 @@ To run only select services, simply specify only those services. For example, to
 --enable rpc
 ```
 
```

Comment:
Instead of the readme change above, can we add a note here in the section discussing the `--enable` option?
```suggestion
__Note: In `--local` mode the `core` service always runs no matter what options are passed, the `friendbot` faucet service runs whenever `horizon` is running, and `horizon` is run when `rpc` is requested so that friendbot is available.__
```